sk-python-flask/sk_python_flask/app.py

```diff
@@ -13,15 +15,15 @@
 
 @app.route("/skills/<skill_name>/functions/<function_name>", methods=["POST"])
 def execute_semantic_function(skill_name, function_name):
+    if sys.platform == "win32" and sys.version_info >= (3, 8, 0):
```

Comment:
This is being fixed here https://github.com/microsoft/semantic-kernel/pull/1416#pullrequestreview-1482530831 and can be removed in the future.